setup.py

```diff
@@ -26,13 +26,13 @@
 
 
 version_file = join(dirname(abspath(__file__)), 'src', 'DatabaseLibrary', 'version.py')
+pkg_vars  = {}
```

Comment:
The project switched from setup.py to pyproject.toml - so changes to this file are not needed anymore.